.github/labels.json

```
@@ -0,0 +1,181 @@
+  },
+  {
+    "color": "f5a342",
+    "description": "Requests, Issues and Changes targeting javadoc and module documentation",
+    "name": "Category: Doc",
+    "aliases": ["documentation"]
+  },
+  {
+    "color": "f5a342",
+    "description": "Requests, Issues and Changes targeting gameplay mechanics and content",
+    "name": "Category: Gameplay Content",
+    "aliases": ["content"]
+  },
+  {
+    "color": "f5a342",
+    "description": "Requests, Issues and Changes targeting performance",
+    "name": "Category: Performance",
+    "aliases": ["performance"]
+  },
+  {
+    "color": "f5a342",
+    "description": "Requests, Issues and Changes targeting security",
+    "name": "Category: Security",
+    "aliases": ["security"]
+  },
+  {
+    "color": "f5a342",
+    "description": "Requests, Issues and Changes targeting tests and quality assurance",
+    "name": "Category: Test/QA"
+  },
+  {
+    "color": "94EC55",
+    "description": "Good for learners that are new to Terasology",
+    "name": "Good First Issue"
+  },
+  {
+    "color": "876314",
+    "description": "Affects aspects not visible in Singleplayer mode only",
+    "name": "Multiplayer"
+  },
+  {
+    "color": "AAd9d2",
+    "description": "Very big effort likely requiring a lot of research and work in many areas across the codebase",
+    "name": "Size: L",
+    "aliases":["epic"]
+  },
+  {
+    "color": "AAd9d2",
+    "description": "Medium-sized effort likely requiring some research and work in multiple areas",
+    "name": "Size: M",
+    "aliases":["mentor size"]
+  },
+  {
+    "color": "AAd9d2",
+    "description": "Small effort likely only affecting a single area and requiring little to no research",
+    "name": "Size: S",
+    "aliases": ["bite-size"]
+  },
+  {
+    "color": "ff7d7d",
+    "description": "Is currently blocked by missing dependencies, pre-requisites, etc.",
+    "name": "Status: Blocked"
+  },
+  {
+    "color": "E99695",
+    "description": "Requires more information by the author on the reported issue or provided changes",
+    "name": "Status: Needs Author Input"
+  },
+  {
+    "color": "E99695",
+    "description": "Requires help discussing a reported issue or provided PR",
+    "name": "Status: Needs Discussion",
+    "aliases": ["help wanted", "support"]
+  },
+  {
+    "color": "E99695",
+    "description": "Requires to be checked for feasibility, reproducability, etc.",
+    "name": "Status: Needs Investigation",
+    "aliases": ["research", "needs info"]
+  },
+  {
+    "color": "E99695",
+    "description": "Requires to be tested in-game",
+    "name": "Status: Needs Testing"
+  },
+  {
+    "color": "DDDDDD",
+    "description": "Wasn't reviewed or continued for a while",
+    "name": "Status: Stale"
+  },
+  {
+    "color": "3b808e",
+    "description": "Requests, Issues and Changes related to software architecture, programming patterns, etc.",
+    "name": "Topic: Architecture",
+    "aliases": ["architecture", "api", "Architecture"]
```

The labels are processed case-insensitive
```suggestion
    "aliases": ["architecture", "api"]
```